db/migrate/20170616211203_add_indices_to_activities.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndicesToActivities < ActiveRecord::Migration[5.0]
+  def change
+    add_index :activities, :subject_id
+    add_index :activities, :subject_type
+    add_index :activities, :user_id
```

Comment:
For future reference, you can add the index as part of your `create_table` migration.

I believe you can do something like:

```ruby
create_table :activities do |t|
  t.integer :user_id, index: true
  # ...
end
```